cmd/snap/cmd_debug_state.go

```diff
@@ -227,6 +234,201 @@ func (c *cmdDebugState) showIsSeeded(st *state.State) error {
 	return nil
 }
 
+// connState is the connection state stored by InterfaceManager
+type connState struct {
```

Comment:
Hm, so this is a copy of `overlord/ifacestate/ifacemgr.go:type ConnectionState struct {` which we will have to keep in sync - I'm not too keen on this, eventually things will drift. Importing `overlord/ifacestate` directly here would work but I think pulls in a lot of unneeded stuff (like the entire snapstate). I wonder if we could have `overlord/ifacestate/structs` (or json or something) as a common package that both places could import to avoid drift? And if we ever want to support `snap debug state --snaps` we could play the same trick?